man/passwd.1.xml

```diff
@@ -488,5 +484,9 @@
 	<refentrytitle>usermod</refentrytitle><manvolnum>8</manvolnum>
       </citerefentry>.
     </para>
+
+    <para>
+	https://xkcd.com/936/
```

Comment:
this link lacks any sort of introduction or explanation

Reply:
Hmm.  Usually references in SEE ALSO, such as references to other pages, are not introduced.  If a user goes to that site, they'll know why it was referenced.  But I'll think of adding some intro to it, especially for those who don't know what XKCD is, and foreseeing a possible future in which it doesn't exist anymore and people forgot what it was.